server/doc/swdesign/plantuml/seq_startup.plantuml

```diff
@@ -42,6 +42,7 @@ tonic_server -> tonic_server ++--: start listening for state change requests
 grpc_server -> grpc_server ++--: start listening for execution requests /'evil laughter'/
 
 state_loader -> state_loader ++: load startup config
+note right: the startup config\nis provided either\nvia config file or the CLI
```

Comment:
The startup config cannot be provided by the CLI. If there was no startup config, a empty state is loaded which can subsequently be modified.